Unit-Tests/CookieStorage_Tests.m

```diff
@@ -16,6 +18,12 @@ @interface CookieStorage_Tests : CBLTestCaseWithDB
 
 @end
 
+
+@interface CBLCookieStorage (Internal)
+- (NSString*) localDatabaseInfoCookiesKey;
```

Comment:
Don't redeclare this; it should be imported from CouchbaseLitePrivate.h or CBLInternal.h.